src/data/resolvers/status.ts

```diff
@@ -51,17 +56,35 @@ export const statusResolvers = ({
         return null;
       }
     },
-
     async isReputationOracleAlive() {
       try {
+        const latestRootHash = await networkClient.getReputationRootHash();
+        const metaColonyAddress = await networkClient.getMetaColony();
+        const colonyClient = await colonyManager.getClient(
+          ClientType.ColonyClient,
+          metaColonyAddress,
+        );
+        const { skillId } = await colonyClient.getDomain(ROOT_DOMAIN_ID);
         const res = await fetch(
-          `${reputationOracleEndpoint}/${networkNameForReputationOracle}`,
+          `${reputationOracleEndpoint}/${networkNameForReputationOracle}/${latestRootHash}/${metaColonyAddress}/${skillId}/${AddressZero}/noProof`,
```

Comment:
This fails with 400 when there is no reputation in the colony (for example when the Motions Extension has not been enabled). That makes the Network Status change to Poor.

Reply:
Realistically, I feel like this case shouldn't happen but yeah, while in the dev it could be misleading.